stdlib/importlib/readers.pyi

```diff
@@ -0,0 +1,60 @@
+# On py311+, things are actually defined in importlib.resources.readers,
+# and re-exported here,
+# but doing it this way leads to less code duplication for us
+
+import pathlib
+import sys
+from _typeshed import Incomplete, StrPath
+from collections.abc import Iterable, Iterator
+from io import BufferedReader
+from typing import NoReturn, TypeVar
+from typing_extensions import Literal, Never, Self
+
+if sys.version_info >= (3, 11):
+    import importlib.resources.abc as abc
+else:
+    import importlib.abc as abc
+
+if sys.version_info >= (3, 10):
+    if sys.version_info >= (3, 11):
+        __all__ = ["FileReader", "ZipReader", "MultiplexedPath", "NamespaceReader"]
+    else:
```

Comment:
Nit:

```suggestion
    if sys.version_info < (3, 11):
```

While `else` is of course correct here, a separate `if` statement makes it clearer that these are unrelated version differences.